pkg/utils/ports.go

```diff
@@ -28,16 +18,9 @@ func IsPortOpen(t *testing.T, port string) bool {
 	return true
 }
 
+// Deprecated: use https://github.com/smartcontractkit/freeport GetOne
 func MustRandomPort(t *testing.T) string {
-	for i := 0; i < 5; i++ {
-		port := GetRandomPort()
-
-		// check port if port is open
-		if IsPortOpen(t, port) {
-			t.Log("found open port: " + port)
-			return port
-		}
-	}
-
+	p := freeport.GetOne(t)
+	strconv.Itoa(p)
```

Comment:
```suggestion
	return strconv.Itoa(p)
```
And remove the panic below - I guess it being there is why this lack of return wasn't flagged?